### DISCUSSION TITLE
VerifyDirectory - how to check files when file names are dynamic?

### DISCUSSION THREAD

**Discussion by Radamonas** [VerifyTests/Verify#1030] [General]
I want to use VerifyDirectory, however I am checking for files which have dynamic names therefore verified and received fails changes after each test run (in my case I am checking files which have date in their name).

Is there an elegant way to have?
```
yyyyMMdd_filename.verified.txt 
yyyyMMdd_filename.received.txt 
```
Or should I rename the files and then execute VerifyDirectory?

**Comment by SimonCropp** [comment]
VerifyDirectory expects file names to be consistent
